drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -161,7 +161,7 @@ function DragAndDropEditBlock(runtime, element, params) {
                             $(this).addClass('hidden');
                             $('.save-button', element).parent()
                                 .removeClass('hidden')
-                                .one('click', function submitForm(e) {
+                                .on('click', function submitForm(e) {
```

Comment:
@itsjeyd FYI important one-letter change: with `one` if submission failed due to validation error, `Save` button stop working, so you've got to cancel and go all over again.